udm/beginner/party.md

```diff
@@ -8,27 +8,29 @@ Party Data Model Overview
 
 Entities
 1. Party
-Description: Represents any entity that can enter into a relationship. This includes individuals, groups, or organizations.
+Description:  The PARTY entity represents a unified model for individuals (PERSON) and organisations (ORGANIZATION), capturing common attributes and classifications, essential for managing business relationships, contracts, and demographic categorizations.
 * Key Attributes: Unique identifier (partyId).
 2. Person
-* Description: A specialization of Party, representing individual human beings.
-* Key Attributes: Inherits partyId, plus attributes like firstName, lastName, birthDate.
+* Description: A specialization of the Party, representing unique characteristics of individual human beings.
+* Key Attributes: Inherits partyId, plus attributes like firstName, lastName, birthDate providing a foundation set of data to identify and characterize each individual uniquely.
 * Relationship with Party: Every Person is a Party, but with additional attributes specific to individuals.
 3. PartyGroup
 * Description: Another specialization of Party, representing collective entities such as companies or organizations.
 * Key Attributes: Inherits partyId, plus attributes like groupName, taxId.
 * Relationship with Party: Every PartyGroup is a Party, but with attributes specific to groups.
 4. Party Roles
 * Function: Defines the role of a Party in a specific context.
+* There is an Entity RoleType which stores the types of Roles available in the data model, we can also add a new role as per the requirement.
```

Comment:
Just a minor change in this statement.
The RoleType entity stores roles available OOTB like carrier, bill-to customer, ship-from vendor, employee.